src/services/GoogleService.tsx

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const OAUTH2_REVOKE_API = "https://accounts.google.com/o/oauth2/revoke";
+
+export const revokeOauth2Token = (token: string) => {
+  const data = new FormData();
+  data.append("xsrfToken", "");
+  data.append("token", token);
+
+  return axios.post(OAUTH2_REVOKE_API, data);
```

Comment:
Please, check what happens in case access token has expired? Google will not let any api call with expired token, but then again, if it is expired it is already invalid so you don't need to revoke it. Just check how app behaves in that case.